develop-lab/solutions/microservice-speaker/src/main/java/org/acme/conference/speaker/Speaker.java

```diff
@@ -41,15 +39,60 @@ public Speaker(String nameFirst, String nameLast, String organization,
         this.uuid = uuid;
   }
 
-  public Speaker(String nameFirst, String nameLast, String organization,
-            String biography, String picture, String twitterHandle) {
-        this(nameFirst, nameLast, organization, biography, picture, twitterHandle,UUID.randomUUID().toString());
-    }
-
   @JsonbCreator
     public Speaker(String nameFirst, String nameLast, String organization,
             String picture, String twitterHandle) {
-        this(nameFirst, nameLast, organization, "Empty bio", picture, twitterHandle);
+        this(nameFirst, nameLast, organization, "Empty bio", picture, twitterHandle, UUID.randomUUID().toString());
+  }
+
+  public String getUuid() {
+    return uuid;
+  }
+  public void setUuid(String uuid) {
+    this.uuid = uuid;
+  }
+
+  public String getOrganization() {
+    return organization;
+  }
+
+  public void setOrganization(String organization) {
+    this.organization = organization;
+  }
+
+  public String getNameFirst() {
+    return nameFirst;
+  }
+  public void setNameFirst(String nameFirst) {
+    this.nameFirst = nameFirst;
+  }
+
+  public String getNameLast() {
+    return nameLast;
+  }
+  public void setNameLast(String nameLast) {
+    this.nameLast = nameLast;
+  }
+
+  public void setBiography(String biography) {
+    this.biography = biography;
+  }
+  public String getBiography() {
+    return biography;
+  }
+
+  public String getPicture() {
+    return picture;
+  }
+  public void setPicture(String picture) {
+    this.picture = picture;
+  }
+
+  public String getTwitterHandle() {
+    return twitterHandle;
+  }
+  public void setTwitterHandle(String twitterHandle) {
+    this.twitterHandle = twitterHandle;
```

Comment:
With Panache getters and setters are not actually needed, just set the entity's properties public.

You can see this doc for reference: https://quarkus.io/guides/hibernate-orm-panache#how-and-why-we-simplify-hibernate-orm-mappings

Reply:
Noted!!